client/dist/styles.css

```diff
@@ -140,4 +243,5 @@ div {
 
 u:hover {
   background-color: lightgreen;
+>>>>>>> 1d52b68eba7950096d6d13cd23bdfe031c4e4485
```

Comment:
delete line 246

Reply:
Same here.  My css file only goes to line 152.  Not sure why this is showing here.